client/src/components/UpcomingEventsCard/UpcomingEventsCard.css

```diff
@@ -0,0 +1,28 @@
+.UpcomingEventsCard {
+    width: 17vw;
+    height: 14vh;
+    border-radius: 3px;
+    background-color: var(--pink);
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
```

Comment:
Is there a CSS variable for this color? It looks a little lighter than the `Info Card, Pink, Small` card's shadow.